src/runtime/components/NuxtLinkLocale.ts

```diff
@@ -0,0 +1,110 @@
+import { useLocalePath } from '#i18n'
+import { defineComponent, computed, defineNuxtLink, h } from '#imports'
+
+import type { PropType } from 'vue'
+import type { RouteLocationRaw } from 'vue-router'
+import type { RawLocation, RouteLocation } from '@intlify/vue-router-bridge'
+
+const NuxtLinkLocale = defineNuxtLink({ componentName: 'NuxtLinkLocale' })
+
+export default defineComponent({
+  name: 'NuxtLinkLocale',
+  props: {
```

Comment:
With this method of implementation, if new props are added or deprecated in NuxtLink, I think we would not be able to maintain compatibility and there would be maintenance to follow 🤔 

@danielroe 
Is there a better way to do… NuxtLink props inherit without hard-coding as follows?

Reply:
I think you can directly spread the props of NuxtLink here. (`NuxtLinkLocale.props`)

Reply:
Oh, I see, NuxtLinke is exported!

When I looked at the code, it didn't look exported to me, which is why I asked you the mention!
https://github.com/nuxt/nuxt/blob/main/packages/nuxt/src/app/components/nuxt-link.ts

Thanks ❤️

Reply:
I somehow forgot it was possible to spread the props of a component, much better this way, thanks! 🚀